azkaban-common/src/main/java/azkaban/Constants.java

```diff
@@ -47,6 +47,9 @@
   public static final int DEFAULT_SSL_PORT_NUMBER = 8443;
   public static final int DEFAULT_JETTY_MAX_THREAD_COUNT = 20;
 
+  // One Schedule's default End Time: 01/01/2050, 00:00:00, UTC
+  public static final long DEFAULT_SCHEDULE_END_EPOCH_TIME = 2524608000000L;
```

Comment:
@chengren311 I can't see too much difference on this change. Since this code is already in production, I would like we don't change this number.